development/compiling/compiling_for_linuxbsd.rst

```diff
@@ -92,12 +92,23 @@ Start a terminal, go to the root dir of the engine source code and type:
 
 ::
 
-    scons -j8 platform=linuxbsd
+    scons -j8 platform=linuxbsd target=release_debug
 
 A good rule of thumb for the ``-j`` (*jobs*) flag, is to have at least as many
 threads compiling Godot as you have cores in your CPU, if not one or two more.
 Feel free to add the ``-j`` option to any SCons command you see below.
 
+You have 3 target options when compiling.
+
+- ``debug``, Performance will be worse and the executable will be larger.
+  However you can debug Godot and it compiles the fastest.
+- ``release_debug``. Godot will have normal performance and file size, and you
+  can debug it.
+- ``release``. Godot will have normal performance and file size, but Godot
+  cannot be debugged.
```

Comment:
The point of `release` is that performance is *higher* that `release_debug`.

Reply:
IMO this should be covered in the Introduction to the buildsystem (if it isn't already), and each "compiling for" page should have a link to the introduction.